andino_firmware/src/motor.h

```diff
@@ -62,47 +62,44 @@
 // CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#ifndef ANDINO_FIRMWARE_SRC_MOTOR_H_
+#define ANDINO_FIRMWARE_SRC_MOTOR_H_
```

Comment:
I wonder, what compiler and cpp version platform or arduino ide use? isn't `pragma once` recommended? Asking from lack of knowledge

Reply:
I did some research and couldn't find any strong reasons for not using `pragma once`, plus I gave it a try and it seems to be doing its job. Changed. Thanks!